cpp/models/abm/parameters.h

```diff
@@ -233,7 +233,19 @@ struct TestParameters {
     double specificity;
 };
 
-struct AntigenTest {
+struct GenericTest {
+    using Type = TestParameters;
+    static constexpr Type get_default()
+    {
+        return Type{0.9, 0.99};
+    }
```

Comment:
We first tried to make the generic test pure virtual (which would be logical since there is no generic tests to practical use). However, when using the line
return !person.get_tested(m_test_type.get_default());
in the run of the test, the compiler wasn't aware of any get_default() function of a generic test. I thought it would know that it can only be instanced by classes that inherit the generic test and therefore have to have this function, since it is required by the pure virtual generic test class. But since that didn't work, we did this. Maybe then the template is the good solution. We should discuss this briefly.